webapp/src/components/ParcelDetailPage/ParcelDetailPage.css

```diff
@@ -40,8 +40,27 @@
   color: var(--gray);
 }
 
-@media (max-width: 768px) {
-  .ParcelDetailPage .parcel-owner-container {
+@media (max-width: 992px) {
+  .ParcelDetailPage .parcel-owner-container, ParcelDetail .parcel-actions-container {
     text-align: left !important;
   }
+
+  .ui.stackable.grid {
+    width: auto;
+    margin-left: 0!important;
+    margin-right: 0!important;
+  }
+  
+  .ui.stackable.grid:not(.vertically)>.row {
+      margin: 0;
+      padding: 0;
+  }
+
+  .ui.grid>.stackable.stackable.row>.column, .ui.stackable.grid>.column.grid>.column, .ui.stackable.grid>.column.row>.column, .ui.stackable.grid>.column:not(.row), .ui.stackable.grid>.row>.column, .ui.stackable.grid>.row>.wide.column, .ui.stackable.grid>.wide.column {
+    width: 100%!important;
+    margin: 0 0!important;
+    -webkit-box-shadow: none!important;
+    box-shadow: none!important;
+    padding: 1rem 1rem!important;
```

Comment:
piggybacking the comment, this selector _can't_ be the only way to do this. No other css file needs to resort to such a big selector.
Perhaps we can change it? I'm not sure what's doing so I can't suggest a better one

Reply:
Same with `.ui.stackable.grid:not(.vertically)>.row` and `.ui.stackable.grid`. Changing how `row`s or `grid`s work is indicative of a problem. They're supposed to be the backbone of the estructure you build on.

Reply:
I copy and past the exact css from `semantic-ui` I will try to remove the importants, buy idk if changing it to what you said @NicoSantangelo is a good idea too

Reply:
@nicoSantangelo idk if this answer your question:

With those css I set stackable the grids at 992px instead of 786px. We need it due the amount if action buttons.